docs/ref/templates/builtins.txt

```diff
@@ -1732,13 +1732,30 @@ example, when the active locale is ``en`` (English):
 ``34232.00`` ``{{ value|floatformat:"-3g" }}`` ``34,232``
 ============ ================================= =============
 
+Output is always localized (independently of the :ttag:`{% localize off %}
+<localize>` tag) unless the argument passed to ``floatformat`` has the ``u``
+suffix, it will force disabling localization. For example, when the active
+locale is ``pl`` (Polish):
+
+============ ================================= =============
+``value``    Template                          Output
+============ ================================= =============
+``34.23234`` ``{{ value|floatformat:"2" }}``   ``34,232``
+``34.23234`` ``{{ value|floatformat:"2u" }}``  ``34.232``
+============ ================================= =============
+
 Using ``floatformat`` with no argument is equivalent to using ``floatformat``
 with an argument of ``-1``.
 
 .. versionchanged:: 3.2
 
     The ``g`` suffix to force grouping by thousand separators was added.
 
+.. versionchanged:: 4.0
+
+    Localization was forced independently from the :setting:`USE_L10N` setting.
+    Use the ``u`` suffix to disable localization.
```

Comment:
_Something_ about this doesn't read/flow correctly to me, but it's too early in the day for me to pick it out. Highlighting so that perhaps someone else can reason it out.

_Maybe_ for consistency with the `g` suffix note above, something like:

`the ``u`` suffix  to force disabling localisation, independent of the USE_L10N setting, was added`

would work? It's still a bit stunted, but consistent (for reference, if I were rewriting both to be consistent, off the top of my head it'd be something like `Support for the ``u`` suffix was added, to disable localisation regardless of the USE_L10N setting`)

YMMV.

Reply:
> would work? It's still a bit stunted, but consistent (for reference, if I were rewriting both to be consistent, off the top of my head it'd be something like `` Support for the ``u`` suffix was added, to disable localisation regardless of the USE_L10N setting ``)

This is not an accurate description on this change. We have here two things, first `floatformat` no longer depends on the `USE_L10N` setting and the output is always localized, second we added `u` to force disabling localization. Maybe two separate notes:

```
.. versionchanged:: 4.0

    ``floatformat`` template filter no longer depends on the
    :setting:`USE_L10N` setting and always returns localized output.

    The ``u`` suffix to force disabling localization was added. 
```